frontend/src/index.tsx

```diff
@@ -39,6 +43,20 @@ const fetchAllReportData = async (langImplementation: string) => {
   return parseImplementationData(loaderData);
 };
 
+const fetchImplementationStatsData = async () => {
```

Comment:
I think sometime soon I'd like to move to some `Client` object which encapsulates this logic that we keep repeating -- i.e. we should have `Client.implementations()`, `Client.report(dialect)`, (and then any future things we add), rather than sprinkling this logic each time we add a file / change the data layout.

But of course we can handle that in a future PR, just mentioning it for awareness.